cg/src/domains/management/stats/stats.service.js

```diff
@@ -51,6 +52,10 @@ class StatsService {
       processors: processorData
     };
   }
+
+  async events(eventType = 'allEvents', fromBlock = 0, toBlock = 'latest', filter) {
+    return await blockchain.getPastEvents(eventType, fromBlock, toBlock, filter);
```

Comment:
if blockchain.getPastEvents return a promise, we don;t need to use async/await in this case

Reply:
I like to use it tho, makes the code clearer about what is a promise and what isn't

Reply:
For instance, we also don't need to return in the last line of a function, but we do so for clarity